File: CyanAlbum/app/src/main/java/app/cyan/cyanalbum/utils/DLog.java
package app.cyan.cyanalbum.utils;

import android.os.Debug;
import android.util.Log;

/**
 * Created by Cyan on 2017/10/28.
 */

public class DLog {
    public static void i(Class clz,String log){
        Log.i("CyanAlbum",clz.getName());
        Log.i("CyanAlbum",log);
    }
}

File: ViewmodeLivedataLifeCycle/app/src/main/java/wc/c/viewmodelivedatalifecycle/DLog.kt
package wc.c.viewmodelivedatalifecycle

import android.os.Debug
import android.util.Log

class DLog {
    companion object {
        fun i(className: String, tag: String, log: String) {
            Log.i(tag, "\n-----------${className}\n${log}\n===============")
        }

        fun i(className: String, log: String) {
            Log.i("VLL", "\n-----------${className}\n${log}\n===============")
        }
    }
}